node/resolvers/catalog/offer.ts

```diff
@@ -8,17 +9,20 @@ const InstallmentsCriteria = {
 
 export const resolvers = {
   Offer: {
-    Installments: ({Installments}: any, {criteria, rates}: any) => {
+    Installments: ({ Installments }: any, { criteria, rates }: any) => {
       if (criteria === InstallmentsCriteria.ALL) {
         return Installments
       }
       const filteredInstallments = !rates
         ? Installments
-        : filter(({InterestRate}) => !InterestRate, Installments)
+        : filter(({ InterestRate }) => !InterestRate, Installments)
 
       const compareFunc = criteria === InstallmentsCriteria.MAX ? gte : lte
       const byNumberOfInstallments = comparator((previous: any, next) => compareFunc(previous.NumberOfInstallments, next.NumberOfInstallments))
       return [head(sort(byNumberOfInstallments, filteredInstallments))]
     },
+    discountHighLights: ({ DiscountHighLight }: any) => {
+      return map(renameKeys({ '<Name>k__BackingField': 'name' }), DiscountHighLight)
```

Comment:
If you create a resolver for the entire list instead of the properly type you're scopping the object change just for this object (offer)

The correct solution could be create a Discount type resolver like:

Discount: {
     name: prop('<Name>k__BackingField'),
},

Then you rename the field '<Name>k__BackingField' for every Discount instance by doing that you centralize all discount key map in only one piece of code